eng/pipelines/libraries/outerloop.yml

```diff
@@ -33,6 +33,7 @@ jobs:
           - Linux_arm64
           - Linux_musl_arm64
       - ${{ if eq(variables['includeOsxOuterloop'], true) }}:        
+        - OSX_arm64
```

Comment:
I should've caught this... I think we don't have a helix queue setup for OSX_arm64, so I guess this broke the libraries outerloop build. I can fix this in a separate PR and we can enable them once we have a queue for it.

Reply:
since we still dont have stable hardware for this we dont expect to enable test runs for osx_arm64 just yet. Can we leave it disabled till we have that?

Reply:
Yeah, I create a PR for that: https://github.com/dotnet/runtime/pull/43242

Reply:
cool.. thanks.

Reply:
> I should've caught this...

@safern Thanks for being genereous, but this was clearly my fault.  I forget there are three main pipelines all the time.  I get the rolling outerloop and the official build pipeline confused and map them to a single pipeline...

Reply:
Yeah, we have so many ymls that it is sometimes hard to get it right and the easier way to add a new flavor is to try and look where we're already building `osx_x64` and just add entries for the new platform there, so confusing for all I think 😄